3.evaluate_model/get_LOIO_probabilities_all_featurespaces_no_retraining.ipynb

```diff
@@ -0,0 +1,1258 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #94.                        C=model.C,
</pre><p>Obviously it works, but I'm surprised you can access the C parameter, since it is not an attribute on the 1.1.1 sklearn logistic regression documentation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/phenotypic_profiling_model/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Interesting! Thanks for digging into this and raising. I confirmed that I am using 1.1.1 (see below), but maybe it is through the saving and/or loading with joblib process that exposes the attribute again? Maybe this process also exposes parameters (C is a parameter in [1.1 docs](https://scikit-learn.org/1.1/modules/generated/sklearn.linear_model.LogisticRegression.html?highlight=logisticregression#sklearn.linear_model.LogisticRegression)) 🤷 

![image](https://github.com/WayScience/phenotypic_profiling_model/assets/7340421/38fb2052-3f6d-416d-8f31-d1540ddd6a96)